### ISSUE TITLE
hide or encrypt file IDs Gdrive

### ISSUE THREAD

**Issue by OryWareKyrie** [Achrou/goindex-theme-acrou#52]
![image](https://user-images.githubusercontent.com/31232184/83462931-de253a80-a442-11ea-8e5c-bb7c6176c2c4.png)
can you modify to not show the file IDs?
thank you very much boss

**Comment by Achrou** [comment]
Of course, it can be changed, but you are not using the program I wrote.

**Comment by OryWareKyrie** [comment]
Exactly, not in this print, I tested yours here and the same thing happened. I just used it to exemplify. I would appreciate it very much if you could change that. great job. yours is much more stylish than the others

**Comment by louanfontenele** [comment]
Same here.

**Comment by louanfontenele** [comment]
> Of course, it can be changed, but you are not using the program I wrote.

Please, change it, because it's not good if someone can take the informations about the drive. Hope you can fix it. Thank ;)

**Comment by Ognisty321** [comment]
@Aicirou 
We can see folders and files id, pls fix it: https://i.imgur.com/RmyQ2WU.png
